lib/auth/auth.go

```diff
@@ -591,6 +592,30 @@ func (s *AuthServer) generateUserCert(req certRequest) (*certs, error) {
 	return &certs{ssh: sshCert, tls: tlsCert}, nil
 }
 
+// CheckKubeCluster validates kubernetes cluster name against known kubernetes
+// clusters. If kc is empty, this validation succeeds (no error returned).
+func CheckKubeCluster(kc string, pg services.ProxyGetter) (string, error) {
+	if kc == "" {
+		return "", nil
+	}
```

Comment:
Nit: seems like a potential future footgun.  Might be preferable to make this an error and have the caller explicitly skip `CheckKubeCluster` in contexts where an empty kube cluster is allowable.

Reply:
Done